cloud/package.json

```diff
@@ -19,15 +19,18 @@
 		"@types/source-map-support": "^0.5.10",
 		"@typescript-eslint/eslint-plugin": "^6.21.0",
 		"@typescript-eslint/parser": "^6.21.0",
+		"aws-cdk": "^2.131.0",
```

Comment:
what makes this a devdependency and not a standard dependency?

Reply:
This is the CLI for running commands, such as "cdk synth", so it's not part of any built artifacts or build process. It's like using jest to run tests, or typescript to compile the code.

In contrast, aws-cdk-lib is used in our scripts that are run to produce the CloudFormation templates, which are then deployed to AWS.